src/popup.html

```diff
@@ -22,18 +48,50 @@ <h1>Scroll Percentage in Tab Title</h1>
       progress, how much you have read and how much is left.
     </p>
 
-    <div>
-      <h2>Supported websites</h2>
-      <ul id="urlsList"></ul>
-    </div>
-
+    <h2>Settings</h2>
     <div>
       <form id="addUrlForm">
-        <input type="text" id="url" placeholder="Enter website url" />
-        <input type="submit" value="Submit" />
+        <fieldset>
+          <legend>Supported websites</legend>
+          <ul id="urlsList"></ul>
+          <label>
+            <input
+              type="url"
+              id="url"
+              placeholder="Enter website url"
+              required
+            />
+            <a href="#" id="useCurrentUrl">(Current Website)</a>
+          </label>
+          <input type="submit" value="Add to list" />
+        </fieldset>
       </form>
     </div>
 
+    <details id="advancedSettings" open>
+      <summary>Advanced</summary>
+      <form id="settingsForm">
+        <fieldset>
+          <legend>Page length calculate method</legend>
+          <label>
+            <input type="radio" name="calcMethod" value="page" required />
+            Full page.
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="calcMethod"
+              value="content"
+              required
+              checked
+            />
+            Content only (Main text block).
+          </label>
```

Comment:
Please change the radio group to a checkbox input field.
Add "Experimental" near the checkbox description.
Set the default state to unchecked as this is an Opt-in feature.